tests/phpt/fatal_error.phpt

```diff
@@ -0,0 +1,40 @@
+--TEST--
+Test catching fatal errors
+--FILE--
+<?php
+
+namespace Raven\Tests;
+
+use PHPUnit\Framework\Assert;
+use Raven\ClientBuilder;
+use Raven\ErrorHandler;
+
+$vendor = __DIR__;
+
+while (!file_exists($vendor . '/vendor')) {
+    $vendor = dirname($vendor);
+}
+
+require $vendor . '/vendor/autoload.php';
+
+$client = ClientBuilder::create([
+    'server' => 'http://public:secret@local.host/1',
+    'send_attempts' => 1,
+])->getClient();
+
+ErrorHandler::register($client);
+
+register_shutdown_function('register_shutdown_function', function () use ($client) {
+    /** @var \Raven\Transport\HttpTransport $transport */
+    $transport = Assert::getObjectAttribute($client, 'transport');
+    $pendingRequests = Assert::getObjectAttribute($transport, 'pendingRequests');
+
+    Assert::assertEmpty($pendingRequests);
+});
```

Comment:
Those asserts can trigger the autoloading. If this is done under PHP 5 inside an error handling context, it fails silently. You should add an `echo` at the end and expect it to be sure that all those assertions are executed correctly.